src/index.js

```diff
@@ -404,7 +414,9 @@ class Dropzone extends React.Component {
         ref={this.setRef}
         aria-disabled={disabled}
       >
-        {this.renderChildren(children, isDragActive, isDragAccept, isDragReject)}
+        <label {...labelAttributes}>
```

Comment:
I'm wondering if `label` should become the main container instead of `div`? Will it have any styling implications?